tests/foreman/ui/test_remoteexecution.py

```diff
@@ -624,26 +624,30 @@ def test_positive_configure_cloud_connector(
 
     template_name = 'Configure Cloud Connector'
     invocation_id = (
-        entities.JobInvocation().search(query={'search': f'description="{template_name}"'})[0].id
+        target_sat.api.JobInvocation()
+        .search(query={'search': f'description="{template_name}"'})[0]
+        .id
     )
     wait_for(
-        lambda: entities.JobInvocation(id=invocation_id).read().status_label
+        lambda: target_sat.api.JobInvocation(id=invocation_id).read().status_label
         in ["succeeded", "failed"],
         timeout="1500s",
     )
 
-    result = JobInvocation.get_output({'id': invocation_id, 'host': target_sat.hostname})
+    result = target_sat.cli.JobInvocation.get_output(
+        {'id': invocation_id, 'host': target_sat.hostname}
+    )
     logger.debug(f"Invocation output>>\n{result}\n<<End of invocation output")
     # if installation fails, it's often due to missing rhscl repo -> print enabled repos
     repolist = target_sat.execute('yum repolist')
     logger.debug(f"Repolist>>\n{repolist}\n<<End of repolist")
-
-    assert entities.JobInvocation(id=invocation_id).read().status == 0
-    assert 'project-receptor.satellite_receptor_installer' in result
+    assert target_sat.api.JobInvocation(id=invocation_id).read().status == 0
```

Comment:
I suggest checking the output for something that specifically shows that RHC was configured, it has occured many times that the status was 0 but actually something failed.